### ISSUE TITLE
Intent Errors (5004) - 28/11/2018

### ISSUE THREAD

**Issue by SME-Issues** [SME-Issues/issues#1321]
|Expression|Result|
|---|---|
| _How are we doing for cash_ |expected intent to be `query_balance` but found `query_payment`|
| _How are we doing for cash_ |expected intent to be `query_balance` but found `query_payment`|
| _How are we doing for cash_ |expected intent to be `query_balance` but found `query_payment`|
| _How are we doing for cash_ |expected intent to be `query_balance` but found `query_payment`|
| _How are we doing for cash_ |expected intent to be `query_balance` but found `query_payment`|
| _give us the position of the club accounts now_ |expected intent to be `query_balance` but found `query_payment`|
| _what did i have in my current account in may_ |expected intent to be `query_balance` but found `query_payment`|
| _what did i have in my savings account back in june_ |expected intent to be `query_balance` but found `query_payment`|
| _when was my charity term deposit last over £500k_ |expected intent to be `query_balance` but found `query_payment`|
| _We need to take a peek at the position of our saving's account_ |expected intent to be `query_balance` but found `query_invoices`|
| _do I have much in savings_ |expected intent to be `query_balance` but found `query_payment`|
| _have we got any of the charity fund left_ |expected intent to be `query_balance` but found `query_payment`|
| _have we got anything left in the charity fund_ |expected intent to be `query_balance` but found `query_payment`|
| _how healthy is my bank balance_ |expected intent to be `query_balance` but found `query_payment`|
| _how much exactly was there in my club and societies account last month_ |expected intent to be `query_balance` but found `query_payment`|
| _what did I have in my savings account last june_ |expected intent to be `query_balance` but found `query_payment`|
| _what kind of balance i got on my charity term deposits account_ |expected intent to be `query_balance` but found `query_payment`|
| _what kinda balance I got on my charity instant access account_ |expected intent to be `query_balance` but found `query_payment`|
| _give me my current account effective balance_ |expected intent to be `query_balance` but found `query_payment`|
| _give me my current account exciting balance_ |expected intent to be `query_balance` but found `query_payment`|
| _give me my current account green balance_ |expected intent to be `query_balance` but found `query_payment`|
| _give me my current account rabid balance_ |expected intent to be `query_balance` but found `query_payment`|
| _i've gotta have the current account comfortable balance_ |expected intent to be `query_balance` but found `query_payment`|
| _i've gotta have the current account effective balance_ |expected intent to be `query_balance` but found `query_payment`|
| _i've gotta have the current account green balance_ |expected intent to be `query_balance` but found `query_payment`|
| _i've gotta have the current account malodorous balance_ |expected intent to be `query_balance` but found `query_payment`|
| _i've gotta have the current account timely balance_ |expected intent to be `query_balance` but found `query_payment`|
| _how much is in my client's accounts_ |expected intent to be `query_balance` but found `query_invoices`|
| _let me know how much is in my client's accounts_ |expected intent to be `query_balance` but found `query_invoices`|
| _at this current moment in time, how much is in my club and society accounts?_ |expected intent to be `query_balance` but found `query_payment`|
| _i want to see the client's account balances_ |expected intent to be `query_balance` but found `query_payment`|
| _is there anything due immediately_ |expected intent to be `query_invoices` but found `query_payment`|
| _is there anything due today_ |expected intent to be `query_invoices` but found `query_payment`|
| _When should I get the money in from ABC ltd?_ |expected intent to be `query_invoices` but found `query_payment`|
| _who should be paid next_ |expected intent to be `query_invoices` but found `query_payment`|
| _am I waiting on any payments_ |expected intent to be `query_invoices` but found `query_payment`|
| _by what date should I pay alpha ltd_ |expected intent to be `query_invoices` but found `query_payment`|
| _does anyone expect payment from me today_ |expected intent to be `query_invoices` but found `query_payment`|
| _have we got much to pay on Tues_ |expected intent to be `query_invoices` but found `query_payment`|
| _how much does my landlord want this month_ |expected intent to be `query_invoices` but found `query_payment`|
| _how much money from me are people waiting on_ |expected intent to be `query_invoices` but found `query_payment`|
| _I'd like to see every sale I've made_ |expected intent to be `query_invoices` but found `query_payment`|
| _report any bills that are due this month_ |expected intent to be `query_invoices` but found `query_payment`|
| _what kind of money is my landlord demanding this month_ |expected intent to be `query_invoices` but found `query_payment`|
| _what will be the value of screw small businesses this year_ |expected intent to be `query_invoices` but found `query_payment`|
| _when is the money from ABC ltd coming in_ |expected intent to be `query_invoices` but found `query_payment`|
| _when will I get the money in from ABC ltd_ |expected intent to be `query_invoices` but found `query_payment`|
| _which bills do you think we should pay this week_ |expected intent to be `query_invoices` but found `query_payment`|
| _which invoices should we pay on Wed_ |expected intent to be `query_invoices` but found `query_payment`|
| _which supplier invoices have yet to be paid_ |expected intent to be `query_invoices` but found `query_payment`|
| _whose payments am I still waiting for_ |expected intent to be `query_invoices` but found `query_payment`|
| _will I have much to pay on Tues_ |expected intent to be `query_invoices` but found `query_payment`|
| _do you know if alpha ltd and beta ltd has paid yet_ |expected intent to be `query_invoices` but found `query_payment`|
| _what will my cash position be after I pay all my invoices._ |expected intent to be `unsupported` but found `query_balance`|
| _how many times has Sam's Deli been late settling their account in the last year_ |Failed because pay delay count query is out of scope. expected intent to be `query_invoices` but found `query_payment`|
| _can you please get us the Aug exciting invoices_ |expected intent to be `query_invoices` but found `query_payment`|
| _can you please get us the Aug sick invoices_ |expected intent to be `query_invoices` but found `query_payment`|
| _can you please get us the Aug sore invoices_ |expected intent to be `query_invoices` but found `query_payment`|
| _can you show me eta bros unpaid comfortable invoices_ |expected intent to be `query_invoices` but found `query_payment`|
| _can you show me eta bros unpaid effective invoices_ |expected intent to be `query_invoices` but found `query_payment`|
| _can you show me eta bros unpaid exciting invoices_ |expected intent to be `query_invoices` but found `query_payment`|
| _can you show me eta bros unpaid green invoices_ |expected intent to be `query_invoices` but found `query_payment`|
| _can you show me eta bros unpaid interstellar invoices_ |expected intent to be `query_invoices` but found `query_payment`|
| _can you show me eta bros unpaid lovely invoices_ |expected intent to be `query_invoices` but found `query_payment`|
| _can you show me eta bros unpaid malodorous invoices_ |expected intent to be `query_invoices` but found `query_payment`|
| _can you show me eta bros unpaid rabid invoices_ |expected intent to be `query_invoices` but found `query_payment`|
| _can you show me eta bros unpaid sick invoices_ |expected intent to be `query_invoices` but found `query_payment`|
| _can you show me eta bros unpaid smelly invoices_ |expected intent to be `query_invoices` but found `query_payment`|
| _can you show me eta bros unpaid sore invoices_ |expected intent to be `query_invoices` but found `query_payment`|
| _can you show me eta bros unpaid timely invoices_ |expected intent to be `query_invoices` but found `query_payment`|
| _who charged me the most this month_ |expected intent to be `query_payment` but found `query_invoices`|
| _which payment is costing me the most money_ |expected intent to be `query_payment` but found `query_balance`|
| _did i receive any payments this week from ikea_ |expected intent to be `query_payment` but found `query_invoices`|
| _did i remember to pay Highfield Ltd on Tuesday_ |expected intent to be `query_payment` but found `query_invoices`|
| _i like to make queries about my payments_ |expected intent to be `query_payment` but found `query_invoices`|
| _are there any paid invoices from john taylor this month_ |expected intent to be `query_payment` but found `query_invoices`|
| _are there going to be any one off payments this year_ |expected intent to be `query_payment` but found `query_invoices`|
| _can you compute the total on invoices paid in the second quarter_ |expected intent to be `query_payment` but found `query_invoices`|
| _check whether I remembered to pay Highfield Ltd on Tuesday_ |expected intent to be `query_payment` but found `query_invoices`|
| _did Highfield Ltd find the time to pay us on Thursday_ |expected intent to be `query_payment` but found `query_invoices`|
| _did Highfield Ltd make the time to pay us on Thursday_ |expected intent to be `query_payment` but found `query_invoices`|
| _did I find the time to pay Gibson Ltd on Monday_ |expected intent to be `query_payment` but found `query_invoices`|
| _did I find the time to pay Rasmussen Ltd this month_ |expected intent to be `query_payment` but found `query_invoices`|
| _did I find the time to pay their bills_ |expected intent to be `query_payment` but found `query_invoices`|
| _did I make time to pay their bills_ |expected intent to be `query_payment` but found `query_invoices`|
| _did I remember to pay Parks Ltd last Wednesday_ |expected intent to be `query_payment` but found `query_invoices`|
| _did I remember to pay Rasmussen Ltd last month_ |expected intent to be `query_payment` but found `query_invoices`|
| _did I remember to pay Rasmussen Ltd this month_ |expected intent to be `query_payment` but found `query_invoices`|
| _Did I Remember To Pay Rasmussen_Ltd_ |expected intent to be `query_payment` but found `query_invoices`|
| _did I tell you to make a payment to Gibson Ltd last month_ |expected intent to be `query_payment` but found `query_invoices`|
| _did I tell you to make any payments to Rasmussen Ltd this week_ |expected intent to be `query_payment` but found `query_invoices`|
| _do we have to make any one off payments last year_ |expected intent to be `query_payment` but found `query_invoices`|
| _find out if I remembered to pay Highfield Ltd on Tuesday_ |expected intent to be `query_payment` but found `query_invoices`|
| _have you got anything on bills paid in the last 4 weeks_ |expected intent to be `query_payment` but found `query_invoices`|
| _How much came through from apple last week_ |expected intent to be `query_payment` but found `query_invoices`|
| _show us all the paid invoices from rich last week_ |expected intent to be `query_payment` but found `query_invoices`|
| _how much did I get accidentally from apple last week_ |expected intent to be `query_payment` but found `query_invoices`|
| _how much did I get eagerly from apple last week_ |expected intent to be `query_payment` but found `query_invoices`|
| _how much did I get fast from apple last week_ |expected intent to be `query_payment` but found `query_invoices`|
| _how much did I get loudly from apple last week_ |expected intent to be `query_payment` but found `query_invoices`|
| _how much did I get quickly from apple last week_ |expected intent to be `query_payment` but found `query_invoices`|
| _how much did I get rarely from apple last week_ |expected intent to be `query_payment` but found `query_invoices`|
| _how much did I get usually from apple last week_ |expected intent to be `query_payment` but found `query_invoices`|
| _show me all my aeroplane_ |expected intent to be `query_payment` but found `query_balance`|
| _how much baldrick did I get from apple last week_ |expected intent to be `query_payment` but found `query_invoices`|
| _how much eagle did I get from apple last week_ |expected intent to be `query_payment` but found `query_invoices`|
| _how much effect did I get from apple last week_ |expected intent to be `query_payment` but found `query_invoices`|
| _how much objective did I get from apple last week_ |expected intent to be `query_payment` but found `query_invoices`|
| _how much puddle did I get from apple last week_ |expected intent to be `query_payment` but found `query_invoices`|
| _how much stick did I get from apple last week_ |expected intent to be `query_payment` but found `query_invoices`|
| _how much watch did I get from apple last week_ |expected intent to be `query_payment` but found `query_invoices`|
| _how much weed did I get from apple last week_ |expected intent to be `query_payment` but found `query_invoices`|
| _how much weeds did I get from apple last week_ |expected intent to be `query_payment` but found `query_invoices`|
| _show me all my objective_ |expected intent to be `query_payment` but found `query_invoices`|
| _how much interest did i receive on my client's accounts_ |expected intent to be `query_payment` but found `query_invoices`|
| _did i receive a payment from Omicron and Sons last month_ |expected intent to be `query_payment` but found `query_invoices`|
| _did i receive a payment from Theta Corp last month_ |expected intent to be `query_payment` but found `query_invoices`|
| _how many payments did i get from Omicron and Sons last month_ |expected intent to be `query_payment` but found `query_invoices`|
| _how much did i get from Omicron and Sons last week_ |expected intent to be `query_payment` but found `query_invoices`|
